modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -128,34 +117,41 @@ In this XML example, the Replay Topic Listener acts like an inbound endpoint for
+You can use Salesforce Connector as an inbound connector to stream data from Salesforce into your application. To use the connector in this capacity, place Salesforce Connector at the start of your flow and do not wrap it in a poll scope.
 
-NOTE: Studio automatically converts the connector to Salesforce (Streaming) mode. Technically, this is still the same connector, but it accesses the Salesforce Streaming API. This means that the only operation the converted connector can perform is *Subscribe Topic Listener* (that is, subscribe to PushTopic).
+Studio automatically converts the connector to Salesforce Streaming mode, which accesses the Salesforce Streaming API. The converted connector can perform only the  *Subscribe Topic Listener* operation in this mode.
 
-image::salesforce/salesforce-studio-subscribe-streaming-channel.png[subscribe streaming channel]
+*Reviewers: Is *Subscribe Topic Listener* correct in the paragraph aboe=ve, or should this be *Subscribe Channel Listener?*
 
-Salesforce connector: Listens to notifications on a topic and feeds the data into the flow.
+In this example, Salesforce Connector listens to notifications on a topic and feeds the data into the flow:
+
+image::salesforce/salesforce-studio-subscribe-streaming-channel.png[subscribe streaming channel]
 
-See also: https://developer.salesforce.com/docs/atlas.en-us.api_streaming.meta/api_streaming/intro_stream.htm[Salesforce Streaming API]
+Streaming channels provide notifications that are not limited to record-based events to subscribers. You can use Salesforce Connector to work with Salesforce streaming channels.
 
-Streaming channels provide notifications to subscribers that are not limited to record-based events. You can use the Salesforce connector to work with Salesforce streaming channels.
+For more information about streaming channels, see the https://developer.salesforce.com/docs/atlas.en-us.api_streaming.meta/api_streaming/intro_stream.htm[Salesforce Streaming API].
 
 [[streamingchannel]]
 == Create a Streaming Channel to Receive Data From Salesforce
 
-You must have the proper Salesforce Streaming API permissions enabled in your organization.
+To create a streaming channel, you must have the proper Salesforce Streaming API permissions enabled in your organization.
 
-. Log into your Salesforce Developer Edition organization.
+Follow these steps to create a streaming channel:
+
+. Log in to your Salesforce Developer Edition organization.
 . Under *All Tabs (+)*, select *Streaming Channels*.
 . On the *Streaming Channels* tab, select *New* to create a new streaming channel.
-. Enter `/u/notifications/ExampleUserChannel` in the *Streaming Channel Name* field, and an optional description.
+. Enter `/u/notifications/ExampleUserChannel` in the *Streaming Channel Name* field.
+. Enter an optional description.
+
+*Reviewers: Where does the next paragraph fit in? Is it a step to perform after step 5 in the previous procedure?*
```

Comment:
The steps above describe how to create a streaming channel in the Salesforce platform. The text below is a note saying that one can also use the connector to create the streaming channel if he wants, by using one of the 2 operations Create or Publish Streaming Channel